### DISCUSSION TITLE
How to reproduce the utils?

### DISCUSSION THREAD

**Discussion by jinhan26** [clarenceluo78/deep-learning-lookup#4] [Q&A] [answered]
Hi, I want to ask how to reproduce the utils for your deep-learning-lookup since I noticed many of your codes are jupyter notebooks? Is there any more straight-forward way to deploy these utility functions?

**Comment by clarenceluo78** [comment, marked as answer]
Hi! As you look closer, you can see some python scripts under the `lib` folder:
<img width="684" alt="截屏2023-03-22 00 53 57" src="https://user-images.githubusercontent.com/71917860/226683892-a4cee84d-a886-455e-af5a-35a3d56d063f.png">
and that's where the actual utility functions stay.

For example, in the `data` file, there are some utility functions related to processing data, transforming format, etc. You can find more details in other files. One other thing that worth NOTICE is that the `env` file contains the config of this python package, so if you wish to reproduce using the codes, remember to modify it.
